CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix `rotate` prop on `Icon` not working in `rtl` @mnajdova ([#1179](https://github.com/stardust-ui/react/pull/1179))
 - `FocusTrapZone` - Do not propagate any keyboard events @sophieH29 ([#1180](https://github.com/stardust-ui/react/pull/1180))
 
+### Features
+- Export `flag` icon in Teams theme @layershifter ([#1189](https://github.com/stardust-ui/react/pull/1189))
```

Comment:
a bit lost here - seems that PR description doesn't correspond to the actual changes introduced. Also, it seems that there is a mistaken item referenced in the PR description as 'fixed' - or it was about something specific from this referenced PR?

Reply:
Thanks, copy paste is evil sometimes, updated 👍